src/main/java/edu/harvard/iq/dataverse/dataaccess/DataAccess.java

```diff
@@ -42,9 +44,16 @@ public DataAccess() {
 
     };
 
+    public static final String FILE = "file";
+    public static final String S3 = "s3";
+    static final String SWIFT = "swift";
+    static final String REMOTE = "remote";
```

Comment:
At some point, should we consider grouping these into an enum?

Reply:
Probably makes sense - I'll try to remember as I look into the larger refactoring I brought up in a recent tech hour.

Reply:
Sounds good. Future refactoring works for me. I just wanted to bring it up while I was looking at it.